.ci/deploy/pantheon/dev-multidev

```diff
@@ -65,4 +65,5 @@ terminus -n env:clear-cache $TERMINUS_SITE.$TERMINUS_ENV </dev/null
 terminus -n build:secrets:set "$TERMINUS_SITE.$TERMINUS_ENV" token "$DEVOPS_TOKEN" --file='devops-secrets.json' --clear --skip-if-empty </dev/null
 
 # Apply upstream updates
+terminus connection:set "$TERMINUS_SITE.$TERMINUS_ENV" git
```

Comment:
This will delete any uncommitted changes from the environment. Is that safe because these are always CI environments?

Reply:
The environment is always being created from scratch in the dev-multidev script, so yes, I believe so.